cmd/ocitools/generate.go

```diff
@@ -382,3 +406,15 @@ func setupLinuxNamespaces(g generate.Generator, needsNewUser bool, nsMaps map[st
 		g.AddOrReplaceLinuxNamespace(nsName, nsPath)
 	}
 }
+
+func onlyExportFlagSpecified(context *cli.Context) bool {
+	onlyExportFlags := []bool{
+		context.Bool("seccomp-only"),
+	}
+	for _, flag := range onlyExportFlags {
+		if flag {
+			return true
```

Comment:
It's a more general approach. If we wanted to add a feature to export just the configuration for one thing then it can be added into this function.